.github/dependabot.yml

```diff
@@ -19,3 +19,15 @@ updates:
       - python
     commit-message:
       prefix: chore(python-deps)
+
+  - package-ecosystem: npm
```

Comment:
can we make it so all these updates result in one PR (maybe one for python and one for node) vs. N different PRs for each dependency bump? right now we get a stream of updates every week, feels a lot. but maybe this is not a good idea.